src/clint.py

```diff
@@ -2314,7 +2314,7 @@ def CheckSpacing(filename, clean_lines, linenum, nesting_state, error):
     # there's too little whitespace, we get concerned.  It's hard to tell,
     # though, so we punt on this one for now.  TODO.
 
-    match = Search(r'(?:[^ (*/![])+(?<!\+\+|--)\*', line)
+    match = Search(r'(?:[^ (*/![]\))+(?<!\+\+|--)\*', line)
```

Comment:
This rule is triggered by `(char *)*fnamep` but adding a space after `)` triggers another rule to remove the space.